### PR TITLE
chore: update demo instructions

### PR DESCRIPTION
**Pull request by rslibed**
update package.json and demo instructions

### CODE REVIEW

File: 2024/calcite-react/demo/src/App.tsx [hunk 1]
```diff
@@ -1,8 +1,12 @@
 import { useState } from 'react';
 
+// Step 6: Import Calcite Shell
 import '@esri/calcite-components/dist/components/calcite-shell';
+
+// Please note: At React 19, we are expecting React to fully support web components which will eliminate the need of web component wrappers, but as of now we will use these.
```
Comment (maxpatiiuk) [inline 2024/calcite-react/demo/src/App.tsx]: (optional)

```suggestion
// Please note: At React 19, we are expecting React to fully support web components which will eliminate the need for web component wrappers, but as of now we will use these.
```

File: 2024/calcite-react/demo/src/App.tsx [hunk 2]
```diff
@@ -29,9 +34,12 @@ function App({ webmap, title, panelHeading }: AppProps) {
       {/* Default content slot */}
       <ArcgisMap
         itemId={webmap}
+        // Step 9: Listen for on view ready
         onViewReady={(e) => {
           const { view } = e.detail;
+          // Disable popup for features widget as feature information will be rendered in side panel
           view.popupEnabled = false;
```
Comment (maxpatiiuk) [inline 2024/calcite-react/demo/src/App.tsx]: should we move this into the useEffect that is in Panel.tsx as this logic better relates to that code (thus should be colocated)

File: 2024/calcite-react/demo/src/main.tsx [hunk 3]
```diff
@@ -1,28 +1,29 @@
...
+import esriConfig from '@arcgis/core/config';
+
 import App from './App.tsx';
 
-// Import CSS
+// Step 1: Import CSS
 import './index.css';
 
-// JSON containing values to configure app
+// Step 3 (OPTIONAL): JSON containing values to configure app
```
Comment (maxpatiiuk) [inline 2024/calcite-react/demo/src/main.tsx]: for an optional step, it shouldn't be screaming too loud 😁

```suggestion
// Step 3 (optional): JSON containing values to configure app
```

also, did you miss step 2?

File: 2024/calcite-react/demo/src/main.tsx [hunk 4]
```diff
@@ -1,28 +1,29 @@
...
   webmap,
   title,
   panelHeading,
 } from './config/application.json';
 
-// Calcite Components Assets (Recommended to use CDN based on documentation)
+// Step 4: Calcite Components Assets (Recommended to use CDN based on documentation)
 // Please note: You can also manage assets locally by copying assets directly from Calcite Components in your project.
 import { setAssetPath } from '@esri/calcite-components/dist/components/index';
 setAssetPath(
```
Comment (maxpatiiuk) [inline 2024/calcite-react/demo/src/main.tsx:22]: is this the default URL or are users required to set it?
if it's default, then should be mark this step as optional?
(or even remove it from the code and just segregate it to the "additional resources" if it's only needed for more advanced setups)

**Review by maxpatiiuk — APPROVED** [review]

File: 2024/calcite-react/demo/src/main.tsx (hunk 4 above)
Comment (rslibed) [reply 2024/calcite-react/demo/src/main.tsx:22]: just tested and looks like it's required to set. I get an error if it isn't set 

image

**Pull request merged**